kfs-core/src/main/resources/edu/arizona/kfs/gl/gl-resources.properties

```diff
@@ -0,0 +1 @@
+error.gl.GlobalTransactionEdit=Accounting line transaction is not allowed because it violates the following Global Transaction Edit validation rule: Reference Origin Code={0}, Fund Group={1}, Sub-Fund Group={2}, Document Type={3}, Object Type={4}, Object Sub-Type={5}.
```

Comment:
Where is the text for the other three GTE error messages (see KFSKeyConstants.java)?